product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -613,3 +491,60 @@ bdr.wait_for_join_completion(verbose_progress boolean DEFAULT false)
+```
+
+### Parameters
+
+| Name | Description |
+| ---- | ------------|
+| `node_group_name` | Name of an existing PGD group. The local node must be part of the group. |
+| `insert_to_update` | Reserved for backward compatibility. |
+| `update_to_insert` | Reserved for backward compatibility. This option is deprecated and will be disabled or removed in future versions of PGD. Use `bdr.alter_node_set_conflict_resolver` instead. |
+| `ignore_redundant_updates` | Reserved for backward compatibility. |
+| `check_full_tuple` | Reserved for backward compatibility. |
+| `apply_delay` | How long nodes wait to apply incoming changes. This is useful mainly to set up a special sub-group with delayed subscriber-only nodes. Don't set this on groups that contain data nodes or on the top-level group. Default is `0s`. |
+| `check_constraints` | Whether the apply process checks the constraints when writing replicated data. It's recommended to keep this to the default value, otherwise you risk data loss. Valid values are either `on` or `off`. Default is `on`. Applies to top-level group only. |
+| `num_writers` | Number of parallel writers for the subscription backing this node group. Valid values are either `-1` or a positive integer. `-1` means the value specified by the GUC `bdr.writers_per_subscription` is used. `-1` is the default. Applies to top-level group only.|
+| `enable_wal_decoder` | Enables/disables the decoding worker process. You can't enable the decoding worker process if `streaming_mode` is already enabled. Valid values are either `on` or `off`. Default is `off`. Applies to top-level group only.|
+| `streaming_mode` | Enables/disables streaming of large transactions. When set to `off`, streaming is disabled. When set to any other value, large transactions are decoded while they're still in progress, and the changes are sent to the downstream. If the value is set to `file`, then the incoming changes of streaming transactions are stored in a file and applied only after the transaction is committed on upstream. If the value is set to `writer`, then the incoming changes are directly sent to one of the writers, if available. If parallel apply is disabled or no writer is free to handle streaming transaction, then the changes are written to a file and applied after the transaction is committed. If the value is set to `auto`, PGD tries to intelligently pick between `file` and `writer`, depending on the transaction property and available resources. You can't enable `streaming_mode` if the WAL decoder is already enabled.<br/><br/>For more details, see [Transaction streaming](../transaction-streaming). Applies to top-level group only.|
+| `default_commit_scope` | The commit scope to use by default, initially the `local` commit scope. This applies only to the top-level node group. You can use individual rules for different origin groups of the same commit scope. See [Origin groups](../durability/commit-scopes/#origin-groups) for more details. |
```

Comment:
I think we can say that this is supported for subgroups as well.  I think the support is not straightforward and would need some more explanation as pointed by Vaijayanti here : [https://enterprisedb.atlassian.net/browse/BDR-4947](https://www.google.com/url?q=https://enterprisedb.atlassian.net/browse/BDR-4947&sa=D&source=editors&ust=1715851822438315&usg=AOvVaw0ZFBALowhZHXAnNTCPdaEb)

Reply:
No it is not in this PR, but someone is going to have to explain to me why I'm being told conflicting things. I was told that the functions in the older bdr.alter_node_group_config( did not work for subgroups.

Now, back up at line 38 is the same statement which I assume is wrong now. 

But someone is going to have to clarify (ie not just say "that is wrong") what the functionality of default_commit_scope is and how it differs between the two calls, assuming that is actually a thing.